mmhealth/converter.go

```diff
@@ -0,0 +1,66 @@
+package mmhealth
+
+import (
+	"github.com/coltoneshaw/mmhealth/mmhealth/types"
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+// convertV1ToV2Diagnostics converts a V1 SupportPacket to V2 SupportPacketDiagnostics
+func convertV1ToV2Diagnostics(v1 types.SupportPacket) model.SupportPacketDiagnostics {
```

Comment:
Should we type alias `model.SupportPacketDiagnostic` as `SupportPacketDiagnosticV2` to make it easier to use?